pages/blog/how-can-i-recover-my-phone.js

```diff
@@ -0,0 +1,336 @@
+import Head from 'next/head'
+import Layout from '../../components/layout'
+
+export default function RecoverPhoneBlog() {
+  return (
+    <Layout>
+      <Head>
+        <title>Recover.ws - Loser Box to protect your item from loss</title>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <div className="recover_phone_container">
+        <section className="hero-section">
+          <h1>How can I Recover my Phone if it is lost?</h1>
```

Comment:
This is the Title (the font should be bigger than H1)

Reply:
Adjusted the font-size of the H1 to make the title bigger.